src/components/like-button/like-button.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
+import { Toggle } from '../toggle/toggle';
+import { HeartIcon } from '../icons';
+
+interface LikeButtonProps {
+	count: number;
```

Comment:
There should be a possiblity on having a isLikedByUser property. the user may have liked already and then it should initially be in the liked state.

Reply:
@malinovic fixed